master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -219,7 +220,7 @@ func (a *agentResourceManager) createResourcePoolSummary(
 			instanceTypeStringBuilder := strings.Builder{}
 			instanceTypeStringBuilder.WriteString(pool.Provider.GCP.InstanceType.MachineType)
 			instanceTypeStringBuilder.WriteString(", ")
-			instanceTypeStringBuilder.WriteString(string(pool.Provider.GCP.InstanceType.GPUNum))
+			instanceTypeStringBuilder.WriteString(fmt.Sprint(pool.Provider.GCP.InstanceType.GPUNum))
```

Comment:
`strconv.Itoa` would be a slightly better fit, I think. Although, looking at the context, it seems like this whole block ought to be replaced by a `fmt.Sprintf("%s, %dx%s", MachineType, GPUNum, GPUType)`.

Reply:
Changed